docs/pubsub-usage.rst

```diff
@@ -20,24 +32,27 @@ Create a new topic for the default project:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> topic = Topic('topic_name')
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
```

Comment:
I think there are two chief issues with doing this

- The name `create_topic` is ambiguous to new users. Does it mean create a `Topic` instance or actual `insert` one?
- We'd need to offer `Client.topic` and `Client.create_topic` side-by-side since users would need to construct pre-existing topics without sending `insert`.